views/layouts/layout-redirect.njk

```diff
@@ -0,0 +1,20 @@
+---
+layout: layout-single-page.njk
+ignore_in_sitemap: true
+---
+
+{% set title = title + ' page has moved' %}
```

Comment:
Can we add a `<meta name="robots" content="noindex">` to the `<head>` for redirected pages?